engine/evarutil.mli

```diff
@@ -226,6 +230,10 @@ type clear_dependency_error =
 
 exception ClearDependencyError of Id.t * clear_dependency_error * GlobRef.t option
 
+(** Marks an evar that has been defined by another evar by projection.
+    Used to handle checking of created goals correctly. *)
+val evar_evar_solution : bool Store.field
```

Comment:
Like the resolvable field, you should use `unit Store.field` to save space on the most common case, which is probably going to be `false`.

Reply:
Done